packages/config/src/layer2s/templates/polygonCDKStack.ts

```diff
@@ -482,6 +493,7 @@ export function polygonCDKStack(templateVars: PolygonCDKStackConfig): Layer2 {
           templateVars.rollupModuleContract.name,
           {
             description: `The main contract of the ${templateVars.display.name}. Contains sequenced transaction batch hashes and forced transaction logic.`,
```

Comment:
I suppose this is taken from Astar but it does not apply to polygon zkEVM or cdk rollups. For Polygon it was "Contains sequencing and forced transaction logic.", we can have this for cdk rollups and  "Contains sequenced transaction batch hashes and forced transaction logic." for validiums as defaults

Reply:
or we can do with "Contains sequencing" for both and be less specific

Reply:
i think it's ok as it is